modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,146 @@
+
+=== RBAC terminology
+
+Under the RBAC framework, you create *roles*, grant *permissions* to those roles, and assign the roles to *users*. When you change the permissions for a given role, all users with that role automatically gain the modified permissions. You grant or deny permissions for a role by creating an ACL and specifying the RBAC role as either allowed or denied  respectively.
+
+Redpanda treats all *users* as security principals and defines them with the `Type:Name` syntax. (eg: `User:mike`) You may omit the `Type` when defining a principal and Redpanda will assume the `User:` type. All examples here use the full syntax for clarity.
+
+See xref:security/authorization/acl.adoc[Access Control Lists] for more information on defining ACLs and working with principals.
+
+==== Roles
+
+A role is a named collection of ACLs which has users assigned to it. You can assign any number of roles to a given user. When installing a new Redpanda cluster, no roles are provisioned by default. If you wish to use RBAC, you will need to create your first roles using your `superuser` account.
+
+See xref:security/authentication.adoc[] for more information on creating and managing superusers.
+
+When performing an upgrade from older versions of Redpanda, all existing SASL/SCRAM users are assigned to the `Users` role. This role has no assigned ACLs. You will need to use your `superuser` account to create additional roles and assign appropriate ACLs as necessary.
+
+==== Policy conflicts
+
+You can assign a combination of ACLs and roles to any given principal. ACLs allow permissions, deny permissions, or specify a combination of both. As a result, users may at times have role assignments with conflicting policies. If this situation arises, the user is permitted to perform an operation if and only if:
+
+* There does not exist a DENY permission matching the operation.
+* There exists an ALLOW permission matching the operation.
+
+=== RBAC Example
+
+Consider a scenario where you have a set of private topics used by your software engineers to publish application update information to users. All your private topics begin with the prefix `private-`. You might create a new `Software Eng.` role to represent the software engineers given permission to write to these private topics. You would then assign the `Software Eng.` role as the `allowed-role` for a new ACL specifying read and write permissions to `private-*`. Using a wildcard includes all existing private topics as well as any new private topics you might create later. You then assign the new role to Mike and Oren, your two software engineers who will write messages to this topic. The RPK commands to accomplish this are:
+
+[,bash]
+----
+rpk security role create 'Software Eng.'
+rpk acl create --operation read --operation write --topic private-* --allow-role 'Software Eng.'
+rpk security role assign 'Software Eng.' --principal User:mike
+rpk security role assign 'Software Eng.' --principal User:oren
```

Comment:
The `--principal` flag accepts multiple values, so you can do:

```
rpk security role assign 'Software Eng.' --principal User:mike,User:oren
``` 

Also, we can document this without the `User:`, rpk will assume the principal type `User:` if no type is provided:
```
rpk security role assign 'Software Eng.' --principal mike,oren
```